### DISCUSSION TITLE
Manifest.json PWA orientation change

### DISCUSSION THREAD

**Discussion by iecxiv** [Ylianst/MeshCentral#6553] [Q&A] [answered]
I want to change orientation from 'portrait' to 'any' in manifest.json when installing android PWA. Can anyone help me?

**Comment by si458** [comment, marked as answer]
This is hard coded, because we customise it, can change it for u, open up a github enhancement request :)

**Reply by iecxiv** [reply]
Created #6554. Thanks

**Reply by si458** [reply]
all done https://github.com/Ylianst/MeshCentral/commit/545bf58e8d809606afec32703d292176ece6a011
one simple line if you wanted to patch/fix it yourself 👍

**Reply by iecxiv** [reply]
Worked has expected. Thanks you!
